src/app/circle-management/circle-choose/circle-choose.component.html

```diff
@@ -6,11 +6,21 @@ <h1 mat-dialog-title *ngIf="title">
 
   <span class="text-smaller text-italic">Circles can be created from your User Dashboard.</span>
 
-  <mat-form-field class="form-control full-width">
+  <!-- <mat-form-field class="form-control full-width">
     <mat-select placeholder="Circle" [formControl]="circleControl">
       <mat-option *ngFor="let circle of userCircles" [value]="circle">{{circle.name}} | {{circle.description}}
       </mat-option>
     </mat-select>
+  </mat-form-field> -->
```

Comment:
Is it important to leave this commented out code in here? 

```suggestion
```

Reply:
I do want to leave it in case this feature gets reverted.